uw-research-computing/gpu-jobs.md

```diff
@@ -363,23 +252,124 @@ option to your submit file:
 ```
 {: .sub}
 
-Another option that allows jobs to run on backfill servers is "wantFlocking" option. Like "Is_Resumable," this option is best used for short or checkpointed jobs, as "wantFlocking" is overloaded and implies "Is_Resumable." However, unlike "Is_Resumable," this option is not constrained to CHTC servers and allows jobs to land in other campus pools that are not CHTC. One caveat to note is that these other pools are not guaranteed to have some CHTC features, like Docker, or CHTC's file staging system. To access flocking, add the "wantFlocking" option to your submit file:
-
-```
-+wantFlocking = true
```

Comment:
I found this useful.  It makes sense to keep OSG instructions on a different page, but if `+wantFlocking` gives access to a unique subset of GPUs should we keep that instruction somewhere?

I see the OSG page gives instructions for flocking as well.  Maybe the solution is the generalize the OSG section here to include the UW Grid and OSG.

Reply:
I made this section about both UW Grid and OSG.